Client/Application/AppDelegate.swift

```diff
@@ -256,23 +256,28 @@ extension AppDelegate: UINavigationControllerDelegate {
 
 extension AppDelegate {
     private func configureCrashReporter() {
-        let mainBundle = NSBundle.mainBundle()
-        let breakpad = BreakpadController.sharedInstance()
+        BreakpadController.sharedInstance().start(true)
+
+        // Add in custom crash-stats keys
+        if profile?.prefs.boolForKey("SendCrashReports") ?? false {
```

Comment:
I think this should be "crashreports.send.always" —

* Exists: has been prompted.
* Is true: always send.
* Is false: never send.
* Is neither true nor false: prompt.

We can add a second pref for number of times prompted or declined at a later date if we choose.